tfhe/src/integer/server_key/radix_parallel/tests_signed/modulus_switch_compression.rs

```diff
@@ -0,0 +1,42 @@
+use crate::integer::gen_keys_radix;
+use crate::integer::tests::create_parametrized_test;
+#[cfg(tarpaulin)]
+use crate::shortint::parameters::coverage_parameters::*;
+use crate::shortint::parameters::*;
+use rand::Rng;
+
+const NB_TESTS: usize = 10;
+
+create_parametrized_test!(modulus_switch_compression_signed);
+
+fn modulus_switch_compression_signed<P>(param: P)
+where
+    P: Into<PBSParameters>,
+{
+    let param = param.into();
+
+    // Modulus switch compression and multi bit PBS are currently not compatible
+    if let PBSParameters::MultiBitPBS(_) = &param {
+        return;
+    }
```

Comment:
prefer updating the macro with parameters as this is what is done elsewhere and avoids thinking tests are passing while they are not run